lib/tools.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable quotes */
 const axios = require("axios").default;
 const DPTLib = require(__dirname + '/knx/src/dptlib'); //todo for the moment
-
+const FORBIDDEN_CHARS = /[^._\-/ :!#$%&()+=@^{}|~\p{Ll}\p{Lu}\p{Nd}]+/gu;
```

Comment:
question: is this documented in iob?

Reply:
Not directly
https://github.com/ioBroker/ioBroker.js-controller/blob/2bedca9238546950bae7682062419cd3ec7954c6/packages/common/lib/common/tools.js#L43

Reply:
ok